jina/serve/runtimes/gateway/request_handling.py

```diff
@@ -185,12 +186,23 @@ async def _load_balance(self, request):
             async with aiohttp.ClientSession() as session:
                 if request.method == 'GET':
                     async with session.get(target_url) as response:
-                        content = await response.read()
-                        return web.Response(
-                            body=content,
+                        # Create a StreamResponse with the same headers and status as the target response
```

Comment:
this also works for getting the OpenAPI.json or forwarding any other GET command?

Reply:
`/openapi.json`, `/docs` and `/redoc` endpoints work as expected